pages/[username]/[collectionId]/[nftId]/index.tsx

```diff
@@ -3,33 +3,56 @@ import { GetServerSideProps } from 'next';
 import GalleryRedirect from 'scenes/_Router/GalleryRedirect';
 import { MetaTagProps } from 'pages/_app';
 import { openGraphMetaTags } from 'utils/openGraphMetaTags';
-import { graphql, useLazyLoadQuery } from 'react-relay';
-import { NftIdQuery } from '__generated__/NftIdQuery.graphql';
-import GalleryRoute from 'scenes/_Router/GalleryRoute';
+import { DecoratedCloseIcon } from 'src/icons/CloseIcon';
+import styled from 'styled-components';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import useKeyDown from 'hooks/useKeyDown';
+import { useCallback } from 'react';
 
 type NftDetailPageProps = MetaTagProps & {
   nftId: string;
   collectionId: string;
 };
 
 export default function NftDetailPage({ collectionId, nftId }: NftDetailPageProps) {
-  const query = useLazyLoadQuery<NftIdQuery>(
-    graphql`
-      query NftIdQuery($nftId: DBID!, $collectionId: DBID!) {
-        ...NftDetailPageFragment
-      }
-    `,
-    { nftId, collectionId }
-  );
+  const {
+    query: { username },
+    push,
+  } = useRouter();
+
+  // the default "back" behavior from the NFT Detail Page
+  // is a redirect to the Collection Page
+  const collectionRoute = `/${username}/${collectionId}`;
+
+  const handleReturnToCollectionPage = useCallback(() => {
+    push(collectionRoute);
+  }, [collectionRoute, push]);
+
+  useKeyDown('Escape', handleReturnToCollectionPage);
 
   if (!nftId) {
     // Something went horribly wrong
     return <GalleryRedirect to="/" />;
   }
 
-  return <GalleryRoute element={<NftDetailPageScene queryRef={query} nftId={nftId} />} />;
+  return (
+    <>
+      <Link href={collectionRoute}>
+        <StyledDecoratedCloseIcon />
+      </Link>
+      <NftDetailPageScene collectionId={collectionId} nftId={nftId} />
```

Comment:
this page now displays a close icon.

<img width="1166" alt="Screen Shot 2022-05-23 at 3 21 34 AM" src="https://user-images.githubusercontent.com/12162433/169765293-9701fdad-bbd3-4131-b9cc-5b87d5b8e4a2.png">